### ISSUE TITLE
chore: Run builds on Windows

### ISSUE THREAD

**Issue by mefellows** [pact-foundation/pact-go#297]
This was disabled in 2.x.x for reasons I can't remember now. This should shake out any issues with `pact-go install` and obviously create more reliability.

**Comment by YOU54F** [comment]
I assume windows and macos builds don't run due to lack of Docker in GitHub actions on those platforms

Appveyor windows build has been failing, I believe it's due to the pact-cli being unable to communicate with docker via localhost.

**Comment by YOU54F** [comment]
I was able to get macos running on arm64 linux and macos via Cirrus CI

```yml
test_task_template: &TEST_TASK_TEMPLATE
  modules_cache:
    fingerprint_script: cat go.sum
    folder: $GOPATH/pkg/mod
  deps_script: make deps
  clean_script: make clean
  bin_script: make bin
  test_script: make test

linux_test_task:
  env:
    matrix:
      - VERSION: 1.17
      - VERSION: 1.18
  name: Tests (Go $VERSION)
  container:
    image: golang:$VERSION
  <<: *TEST_TASK_TEMPLATE

linux_arm64_test_task:
  env:
    matrix:
      - VERSION: 1.17
      - VERSION: 1.18
  name: Tests (Go $VERSION)
  arm_container:
    image: golang:$VERSION
  <<: *TEST_TASK_TEMPLATE

macos_arm64_test_task:
  macos_instance:
    image: ghcr.io/cirruslabs/macos-ventura-base:latest
  pre_req_script: brew install gox
  <<: *TEST_TASK_TEMPLATE

lint_task:
  name: GolangCI Lint
  container:
    image: golangci/golangci-lint:latest
  run_script: golangci-lint run -v --out-format json > lint-report.json
  always:
    golangci_artifacts:
      path: lint-report.json
      type: text/json
      format: golangci
```

**Comment by mefellows** [comment]
Nice one - thanks! I can't recall the specifics as to why now, but you could well be right.

**Comment by YOU54F** [comment]
Fixed by #443
